packages/eas-cli/src/credentials/ios/appstore/provisioningProfile.ts

```diff
@@ -57,6 +142,15 @@ export async function listProvisioningProfilesAsync(
 ): Promise<ProvisioningProfileStoreInfo[]> {
   const spinner = ora(`Getting Provisioning Profiles from Apple...`).start();
   try {
+    if (USE_APPLE_UTILS) {
+      let profiles = await getProfilesForBundleId(bundleIdentifier);
+      const type = ctx.team.inHouse ? ProfileType.IOS_APP_INHOUSE : ProfileType.IOS_APP_STORE;
+
+      profiles = profiles.filter(profile => profile.attributes.profileType === type);
```

Comment:
```suggestion
      const profilesOfType (or whatever sounds better in english) = profiles.filter(profile => profile.attributes.profileType === type);
```
Please don't reassign variables. It makes code hard to read and doesn't bring any value other than reducing the number of variables (but it doesn't make sense...).